### DISCUSSION TITLE
Still not creating a SSL cert for my domain. Vercel security failing?

### DISCUSSION THREAD

**Discussion by betterbrand** [vercel/community#2004] [Help]
### Summary

Set up a domain yesterday and the SSL is still not provisioned. Screen shows a spinning wheel for over 24 hours. My client is getting very restless.  Asking me why I chose Vercel and I'm running out of reasons.  I sent in a ticket yesterday morning and again this morning.  

Is there a systematic unresolved security issue on the platform?  That's press-worthy, no?

**Comment by smaeda-ks** [comment]
Hi there, I've updated your support ticket. The certificate has been issued. We're sorry for the trouble.

https://www.vercel-status.com/incidents/ktnq1hq3x23x
